src/components/FileDrop.vue

```diff
@@ -9,18 +9,19 @@
 				@change="$emit('getFont', $event)"
 			/>
 			<div class="info">
-				"What can my font do?"
-				<strong>Drop a font</strong>
-				and find out!
+				<strong>Drop a font!</strong>
 				<button
 					type="button"
 					class="button on"
 					@click="$emit('getExampleFont', 'GimletVariable-VF.woff2')"
 				>
-					Try with Gimlet!
+					Try with Gimlet
 				</button>
 				<span class="errormessage" :class="{ show: error }">
-					Oops! I couldn't handle that file.
+					<strong>Oops! I couldn't handle that file.</strong>
+					<a href="mailto:brokenfonts@pixelambacht.nl"
```

Comment:
You can even [populate subject and body](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Creating_links#email_links) if you want.

https://mailtolink.me/?ref=tiny-helpers via https://tiny-helpers.dev/email/ might come in handy (they lack the `?` before subject from what I can see)